packages/docusaurus-theme-classic/src/validateThemeConfig.ts

```diff
@@ -264,9 +264,9 @@ const ThemeConfigSchema = Joi.object({
   colorMode: ColorModeSchema,
   image: Joi.string(),
   docs: DocsSchema,
-  metadatas: Joi.array()
+  metadata: Joi.array()
```

Comment:
Perhaps we should add such a message for users who have used `metadatas`?

```suggestion
  metadatas: Joi.any().forbidden().messages({
    'any.unknown':
      'themeConfig.metadatas has been renamed as themeConfig.metadata',
  }),
```

Reply:
Yes, I will do that. Thanks for reminding